LedService/app/src/main/java/com/bksnake/ledservice/LedControlService.java

```diff
@@ -29,4 +29,13 @@ public IBinder onBind(Intent intent) {
         Log.d(LOG_TAG, "-> onBind()");
         return mBinder;
     }
+
+    @Override
+    public void onDestroy() {
+        Log.d(LOG_TAG, "-> onDestroy()");
+        super.onDestroy();
```

Comment:
It should be called at the end of function.

Reply:
@AksonovSergei  Fix